TSC-Charter.md

```diff
@@ -173,8 +173,10 @@ either a closing vote or a vote to table the issue to the next meeting.
 The call for a vote must be seconded by a majority of the TSC or else the
 discussion will continue.
 
-For all votes, a simple majority of all TSC members for, or against, the issue
-wins. A TSC member may choose to participate in any vote through abstention.
+A proposal passes if and only if more than 50% of the TSC members (excluding
+individuals who explicitly abstain) vote in favor. For example, if there are 20
+TSC members, and 5 of those members indicate that they abstain, then 8 votes in
+favor are required for a resolution to pass.
```

Comment:
This is a charter change and so will require board approval, right? If so and if you want to avoid that, I think it can be left as it was. While not as explicit, I think a reasonable interpretation of the existing text is to be the same as the updated text.

(Another option is to include an Addendum document that includes this sort of greater detail/interpretation of the charter.)